t/basic.t

```diff
@@ -20,6 +20,7 @@ use Test::Warnings;
 use FindBin;
 use lib "$FindBin::Bin/lib";
 use OpenQA::Test::Database;
+use OpenQA::Test::TimeLimit '10';
```

Comment:
I reckon this or a variant of it with a higher timeout should be added to `OpenQA::Test::Case`? Otherwise it means we need to add it to virtual all tests by hand.

I'm just guessing, though. The PR has no description.

Reply:
Added a description.

I suggest we gather experiences with explicit mentions first. If we like the idea we can extend the generic test base class. We only need to be a bit careful as it is less explicit. However I do not see a problem with defining a conservative, rather high limit. Also when that limit is below the limit within our CI system I think the feedback is even more explicit as it comes from our tests and not the external CI harness.

Reply:
Ack. Maybe different defaults for e.g. UI tests also makes sense. Just the idea of having loads of different settings per file made me a bit uncomfortable at first...